Changes.md

```diff
@@ -1,3 +1,17 @@
+1.2.0.0ax (relative to 1.2.0.0a2)
```

Comment:
We released `1.2.0.0a3` already, so this will need moving into a new section. Changes.md conflicts like that are a recurring problem, which I hope to do something about when I can find some time.